lms/security.py

```diff
@@ -87,7 +99,12 @@ def authenticated_userid(self, request):
         return identity.userid
 
     def identity(self, request):
-        lti_user = self._get_lti_user(request)
+        lti_user = None
```

Comment:
I think this line is unnecessary now? `self._get_lti_user()` will either return an `LTIUser`, return `None`, or raise `ValidationError` (in which case we early return).